src/main/java/com/auth0/json/mgmt/jobs/UsersExport.java

```diff
@@ -0,0 +1,74 @@
+package com.auth0.json.mgmt.jobs;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Class that represents an Auth0 Users Export object. Related to the {@link com.auth0.client.mgmt.JobsEntity} entity.
+ */
+@SuppressWarnings({"unused", "WeakerAccess"})
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class UsersExport {
+
+    @JsonProperty("connection_id")
+    private String connectionId;
+    @JsonProperty("format")
+    private String format;
+    @JsonProperty("limit")
+    private int limit;
+    @JsonProperty("fields")
+    private List<List<Map<String, String>>> fields = null;
```

Comment:
I realize this structure is a bit yucky...  This is due to the spec provided by Auth0's API docs; the sample JSON response has an array within an array, which in turn contains a collection of objects that represent the field information.